### ISSUE TITLE
Diffie-Hellman Key Exchange

### ISSUE THREAD

**Issue by shrinidhihegde3** [shrinidhihegde3/Snippify-Hub#62]
A key exchange algorithm that allows two parties to securely share a secret key over a public communication channel.
Add input and output field so that people can experiment

**Comment by SanskariWolf** [comment]
Can you please assign this issue to me? I would like to work over it.

**Comment by shrinidhihegde3** [comment]
Contribute inside Ciphers folder
Add how to use, how it works detailed explaination, Reference, Use case, Also add space and time complexity

**Comment by shrinidhihegde3** [comment]
any updates?

**Comment by SanskariWolf** [comment]
Yup, i am nearly done with the work, will be creating the merge request by today.
